administrator/components/com_associations/helpers/associations.php

```diff
@@ -85,7 +85,8 @@ public static function getComponentProperties($key = '')
 				// For categories.
 				else
 				{
-					$cp[$key]->associations->gethelper->class  = ucfirst(substr($cp[$key]->realcomponent, 4)) . 'HelperAssociation';
+					$catExtension = ucfirst(substr($cp[$key]->extension, 4));
+					$cp[$key]->associations->gethelper->class  = $catExtension . 'HelperAssociation';
 					$cp[$key]->associations->gethelper->method = 'getCategoryAssociations';
```

Comment:
I tested witout this change and works fine in my test.
so please remove this change.